src/Main.elm

```diff
@@ -108,52 +170,69 @@ updateModelFromWebsocketMsg msg model =
                             |> Maybe.withDefault "0"
                     )
     in
-    case ( command, maybeValue ) of
-        ( "R", Just value ) ->
-            ColourPicker.updateColour ColourPicker.Red value model
+    case model of
+        Syncing syncingStatus ->
+            syncingStatus
+                |> updateFromSyncingStatus command maybeValue
+                |> (\ss ->
+                        case ss of
```

Comment:
Lift this case statement into a top level function and call it here to avoid the `\ss` binding?